specification/managedservices/resource-manager/preview/2018-06-01-preview/managedservices.json

```diff
@@ -0,0 +1,856 @@
+            "description": "OK - Returns the complete registration definition with plan details.",
+            "schema": {
+              "$ref": "#/definitions/RegistrationDefinition"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get Registration Definition": {
+            "$ref": "./examples/GetRegistrationDefinition.json"
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "RegistrationDefinitions"
+        ],
+        "operationId": "RegistrationDefinitions_Delete",
+        "description": "Deletes the registration definition.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/RegistratonDefinitionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/ScopeParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK - The registration definition is deleted.",
+            "schema": {
+              "$ref": "#/definitions/RegistrationDefinition"
+            }
+          },
+          "204": {
+            "description": "No Content- The registration definition does not exist."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Delete Registration Definition": {
+            "$ref": "./examples/DeleteRegistrationDefinition.json"
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "RegistrationDefinitions"
+        ],
+        "operationId": "RegistrationDefinitions_CreateOrUpdate",
+        "description": "Creates or updates a registration definition.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/RegistratonDefinitionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/ScopeParameter"
+          },
+          {
+            "name": "requestBody",
+            "in": "body",
+            "required": true,
+            "description": "The parameters required to create new registration definition.",
+            "schema": {
+              "$ref": "#/definitions/RegistrationDefinition"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Ok - Returns information about the updated registration definition.",
+            "schema": {
+              "$ref": "#/definitions/RegistrationDefinition"
+            }
+          },
+          "201": {
```

Comment:
Is this operation a long running operation? When would the API return 201?

Reply:
No, this is not a long running opertation. It returns 201 when created and 200 when updated.